packages/report-server/src/reportBuilder/functions/math.ts

```diff
@@ -3,21 +3,45 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
-import { sum as mathjsSum, divide as mathjsDivide, Matrix, filter } from 'mathjs';
-
+import { sum as mathjsSum, divide as mathjsDivide } from 'mathjs';
+
+const isNumeric = (value: unknown): value is number | undefined =>
+  value === undefined || typeof value === 'number';
+const isNumericOrArray = (
+  value: unknown,
+): value is number | undefined | Array<number | undefined> =>
+  isNumeric(value) || (Array.isArray(value) && value.every(isNumeric));
 const isDefined = <T>(value: T): value is Exclude<T, undefined> => value !== undefined;
 
-export const sum = (numbers?: Matrix): number | undefined => {
-  if (numbers === undefined) {
+export const sum = (
+  ...args: (number | undefined | (number | undefined)[])[]
```

Comment:
Arguably we should make the input type `unknown` at this point, since we're now doing validation on the input so there's no need to specify its type. However, I still find it rather useful to have the type definition here, both for readability and to ensure that whenever we call this function directly in code we still get compile time type checking 👍